odoo/tools/translate.py

```diff
@@ -1470,28 +1470,33 @@ def load_language(cr, lang):
     installer.lang_install()
 
 
+
+def get_po_paths(module_name: str, lang: str):
+    lang_base = lang.split('_')[0]
+    if lang_base == 'es' and lang != 'es_ES':
+        # force es_419 as fallback language for the spanish variations
+        if lang == 'es_419':
+            langs = ['es_419']
+        else:
+            langs = ['es_419', lang]
```

Comment:
Note that the previous code would also load es. It was probably not needed, though.

Reply:
@oomsveta I would even say it's not wanted, we are more in a split approach with this PR